prime-router/SFTP-SETUP.md

```diff
@@ -0,0 +1,64 @@
+# Setting up SFTP
```

Comment:
+1 For creating a doc. Can you reference this doc in the getting started.md doc?